rest_api_demo/app.py

```diff
@@ -8,7 +8,7 @@
 from rest_api_demo.database import db
 
 app = Flask(__name__)
-logging.config.fileConfig('logging.conf')
+logging.config.fileConfig('../logging.conf')
```

Comment:
The problem with these paths are that they are both relative to the current working directory (`pwd`) from which the app was started. The real solution to this problem would be to use something like:

```python
logging_conf_path = os.path.normpath(os.path.join(os.path.dirname(__file__), '../logging.conf'))
logging.config.fileConfig(logging_conf_path)
```

More verbose, but more specific.